src/b3d/chisight/gen3d/pixel_kernels/pixel_rgbd_kernels.py

```diff
@@ -27,69 +27,31 @@ def is_unexplained(latent_value: FloatArray) -> bool:
 
 
 @Pytree.dataclass
-class PixelRGBDDistribution(genjax.ExactDensity):
```

Comment:
I removed this abstraction since we only have 1 implementation of this at the moment.  We can bring it back if we want to have more implementations of it later.